os-autoinst-openvswitch

```diff
@@ -33,6 +37,10 @@ sub init_switch {
     die "can't parse bridge local port MAC" unless $self->{MAC};
     die "can't parse bridge local port IP"  unless $self->{IP};
 
+    ## coolo and richard terrible hack to make tap devices
+    for my $i (0..10) {
+      system("ovs-vsctl add-port $self->{BRIDGE} tap$i tag=999");
+    }
 
```

Comment:
We will need more tap devices to support multiple networks: tap64, tap65, ... and tap128, tap129...
See https://github.com/os-autoinst/os-autoinst/blob/master/backend/qemu.pm#L366
This is probably a good opportunity to add them.